Cargo.toml

```diff
@@ -4,5 +4,6 @@ panic = 'unwind'
 [workspace]
 members = [
     'node',
+    'primitives',
```

Comment:
This is in order that clippy can reach there?

Reply:
> This is in order that clippy can reach there?

yes and fmt too, they rely on workspace members